models/public/retinaface-anti-cov/retinaface-anti-cov.md

```diff
@@ -0,0 +1,112 @@
+# retinaface-anti-cov
+
+## Use Case and High-Level Description
+RetinaFace-Anti-Cov is a customized one stage face detector to help people protect themselves from CovID-19. More details provided in the [paper](https://arxiv.org/abs/1905.00641) and [repository](https://github.com/deepinsight/insightface/tree/master/RetinaFaceAntiCov)
+
+## Specification
+
+| Metric                                                        | Value                                                |
+|---------------------------------------------------------------|------------------------------------------------------|
+| Type                                                          | Object detection, object attributes, facial landmarks|
+| GFLOPs                                                        | 2.7781                                               |
+| MParams                                                       | 0.5955                                               |
+| Source framework                                              | MXNet\*                                              |
+
+## Input
+
+### Original model:
+Image, name - `data` , shape - [1x3x640x640], format [BxCxHxW],
+   where:
+
+    - B - batch size
+    - C - number of channels
+    - H - image height
+    - W - image width
+
+   Expected color order - RGB.
```

Comment:
This should not be indented.

Reply:
Done

Reply:
This also applies to the list (in fact, it _mainly_ applies to the list, because the extra indentation wrongfully turns it into a code block).

Reply:
Done